src/gtfs/read.rs

```diff
@@ -55,10 +55,15 @@ fn get_agency_id(route: &Route, networks: &CollectionWithId<objects::Network>) -
 
 impl From<Agency> for objects::Network {
     fn from(agency: Agency) -> objects::Network {
+        let mut codes = KeysValues::default();
+        codes.insert((
+            "source".to_string(),
+            agency.id.as_ref().cloned().unwrap_or("1".to_string()),
```

Comment:
```suggestion
            agency.id.clone().unwrap_or("1".to_string()),
```

Reply:
If the rule for the `agency_id` is the same than for the `source` `object_code`, then I can just reuse the `id` directly. This line will disappear.